Soomla/Assets/Plugins/Soomla/Profile/social-providers/FBSocialProvider.cs

```diff
@@ -161,18 +186,55 @@ public FBSocialProvider ()
 		/// <param name="success">Callback function that is called if the story update was successful.</param>
 		/// <param name="fail">Callback function that is called if the story update failed.</param>
 		/// <param name="cancel">Callback function that is called if the story update was cancelled.</param>
-		public override void UpdateStory(string message, string name, string caption,
+		public override void UpdateStory(string message, string name, string caption, string description,
 		                                 string link, string pictureUrl, SocialActionSuccess success, SocialActionFailed fail, SocialActionCancel cancel) {
-
-//			checkPermission("publish_actions", ()=> {
-				FB.Feed(
-					link: link,
-					linkName: name,
-					linkCaption: caption,
-					linkDescription: message,
-					picture: pictureUrl,
-					callback: (FBResult result) => {
+			checkPermission("publish_actions", ()=> {
```

Comment:
@vedi, this code is calling share dialog to make posting, so it moved at `UpdateStoryDialog`, and `UpdateStory` code re-writed to posting without any user confirmations, and it requires `publish_actions` permission.
I did it according to UserProfile aligning (iOS and Android has the same methods, which can post from dialog or directly).